src/opnsense/scripts/shell/banner.php

```diff
@@ -122,5 +122,20 @@
         );
     }
 }
+printf("\n\n");
+
+foreach (glob("/conf/sshd/ssh_host_*_key.pub") as $ssh_host_pub_key_file_path) {
```

Comment:
Need to check for enabled SSH and HTTPS :)

Reply:
Great idea! Fixed in this PR and in https://github.com/opnsense/core/pull/2476